main.go

```diff
@@ -45,7 +46,8 @@ func main() {
 func runApp(c *cli.Context) {
 	println("Running!")
 	porto := "amqp://"
-	uri := porto + c.String("user") + ":" + c.String("password") + "@" + c.String("server") + ":" + c.String("port")
+	uri := porto + c.String("user") + ":" + c.String("password") + "@" + c.String("server") +
+		":" + c.String("port") + "/" + c.String("vhost")
```

Comment:
According to section 2 of the rabbitmq URI spec:

>The vhost component may be absent; this is indicated by the lack of a "/" character following the amqp_authority. An absent vhost component is not equivalent to an empty (i.e. zero-length) vhost name.

I'm concerned that adding the `/` character here might lead to some weird behavior in an edge case. It might be better to see if the vhost parameter starts with `/` and add the `/` if not, then concatenate that string onto the rest of the uri.